### ISSUE TITLE
Gephi 0.10.1 | Image Preview plugin

### ISSUE THREAD

**Issue by iamghebs** [gephi/gephi#2764]
## Current Behavior
The Image Preview plugin is missing from Gephi 0.10.1

## Context

The plugin seems to be missing when upgrading from 0.9.x to 0.10.x

## Your Environment

* Version used: Gephi 0.10.1
* Operating System: MacOS Ventura 13.2.1

**Comment by careerhops** [comment]
Yes, even I tried to find the Image Preview plugin, but it is not available in version 0.10. When can we expect it to available or are there an alternate way to access the Image Preview plugin.

**Comment by careerhops** [comment]
Hi, any suggestion how to resolve this issue

**Comment by totetmatt** [comment]
Issue is due to breaking change from gephi 0.9 to gephi 0.10 that needs to be adapted on the plugin:

https://github.com/gephi/gephi-plugins/tree/image-preview-plugin
